gcp-observability/src/main/java/io/grpc/gcp/observability/ObservabilityConfigImpl.java

```diff
@@ -76,6 +100,20 @@ private void parseLoggingConfig(Map<String, ?> loggingConfig) {
         }
         this.eventTypes = eventTypesBuilder.build();
       }
+      String sampler = JsonUtil.getString(config, "global_trace_sampler");
+      Double samplingRate = JsonUtil.getNumberAsDouble(config, "global_trace_sampling_rate");
+      checkArgument(
+          sampler == null || samplingRate == null,
+          "only one of 'global_trace_sampler' or 'global_trace_sampling_rate' can be specified");
```

Comment:
This would also trigger if neither one of them was set.

Reply:
Not true. When neither is set both terms are true and the expression is true. Also all existing tests (without sampler or samplingRate) would break

Reply:
My bad!